app/controllers/channels/base_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Channels::BaseController < ApplicationController
   skip_before_filter :set_locale
+  helper_method :channel_decorator
   before_filter do
     I18n.locale = 'pt'
   end
+
+  def channel_decorator
+    channel.decorator
```

Comment:
tipo não me interessa, se é decorator, presenter, abacaxi, etc
se se comporta como deve isso q é, 
ruby way: se vc tem uma lista de usuarios vc não chama a variavel de `user_list`, só chama de `users` esqueci o nome dessa notação, mas até por isso q o padrão do decorator vc tem geralmente algo como  `@user = User.find(x).decorate` vc atribui para `@user`

a respeito do `helper_method: channel` eu não vejo problema, o channel em si ja não é visivel na view, se for nem precisa desse helper_method